ui/react-components/components/AddAppointment/AddAppointment.jsx

```diff
@@ -790,15 +793,25 @@ const AddAppointment = props => {
                                     if(date.length > 0) {
                                         const selectedDate = moment(date[0]).toDate();
                                         updateAppointmentDetails({appointmentDate: selectedDate});
+
+                                        if (holidays) {
+                                            const formattedHolidays = holidays.replace(/\s+/g, '').split(',');
```

Comment:
Since this formatting step is repetitive for all date pickers, Can we have this formatted in the container which by sending the formatted dates to Add and Edit components?